src/api/app.py

```
@@ -1,50 +1,64 @@
-from flask import Flask, request, jsonify
-import numpy as np
-import cv2
+from flask_socketio import SocketIO, emit
```

Ruff (0.12.2)

`flask_socketio.emit` imported but unused

Remove unused import: `flask_socketio.emit`

(F401)